lib/model/unittest/CMetricModelTest.cc

```diff
@@ -1140,8 +997,8 @@ BOOST_FIXTURE_TEST_CASE(testInfluence, CTestFixture) {
 
         core_t::TTime time{startTime};
         for (std::size_t i = 0u; i < values.size(); ++i) {
-            processBucket(time, bucketLength, values[i], influencers[i], *gatherer,
-                          m_ResourceMonitor, model, annotatedProbability);
+            processBucket(time, bucketLength, values[i], influencers[i],
```

Comment:
For consistency it feels like all the calls to `processBucket` should be qualified (it's our usual style and you've done it for the other methods you've added):
```suggestion
            this->processBucket(time, bucketLength, values[i], influencers[i],
```